src/client-apps/nuxeo-mobile/how-to-customize-nuxeo-mobile-workflow-tasks-layout.md

```diff
@@ -0,0 +1,333 @@
+    name: HOW_TO_workflows_studio_create_json.png
+    1.13#screenshot#up_to_date
+--}}
+![HOW_TO_workflows_studio_create_json](nx_asset://ae86da20-cd15-4327-b2b6-a89b4c081f48 ?w=400)
+
+
+
+Note that the name of this file should match the task .html file name.
+
+Add the following content to this file to enable the task to display the directory selector : 
+```json
+{
+  "name": "nuxeo-task879",
+  "layout": [
+    {
+      "metadata": [
+        {
+          "label": "Type of contract",
+          "field": "type",
+          "type": "directory",
+          "properties": {
+            "multivalued": false,
+            "required": true,
+            "placeholder": "Select the contract type...",
+            "directoryName": "type",
+            "type":"select"
+            
+          }
+        }
+      ]
+    }
+  ]
+}
+```
+
+Save then deploy the package.
+
+### Test your configuration on mobile
+Open Nuxeo Mobile app and set your local url. (can be found on ip tools according to your OS. ex : 172.16.32.75:8080)
+
+Login as administrator ; Assuming you created a document of type “contract” (again, the previous tutorial should be completed), go to this document on mobile and click on the overflow button > start process.
+
+A task is affected to the Administrator user. When opening this task, it should look likes this :
+
+{{!--     ### nx_asset ###
+    path: /default-domain/workspaces/Product Management/Documentation/Documentation Screenshots/CLIENT-APPS/Mobile/HOW_TO_workflows_task_result
+    name: HOW_TO_workflows_task_result.png
+    1.13#screenshot#up_to_date
+--}}
+![HOW_TO_workflows_task_result](nx_asset://5c41d39e-45bd-4272-aa46-58113d4dd1f6 ?w=500)
+
+## Properties list 
+
+Properties can be added depending on the task type :
+
+#### For every tasks type
+```json
+“readonly”:[true/false]
+```
+Used to display informations without letting the user change its value.
+
+
+
+```json
+“required” : [true, false] 
+```
+the “required” properties makes the task label appended with an asterisk (*) and requires this field to be filled for the task to be completed.
+
+
+
+#### For any of tasks user, group, usergroup, directory, document
+
+```json
+“multivalued” : [true, false] 
+```
+If set to true, the user can choose more than one document, user or value
+
+
+
+```json
+“selectedTitle”: string
+```
+Allows to define the selection screen title
+
+
+
+```json
+“type” : [suggest, suggestAll, select]
+```
+- For a simple selection (without search capabilities) the “select” type will make this component a selection switch.
+- suggest :  Alllows to search a user, group, etc.
+- suggestAll : Same as suggest but initially displays the whole set of entries.
+
+```json
+“suggestionPlaceholder”: string
+```
+Enable to customize the placeholder visible in the search bar
+
+
+
+#### For directory type
+```json
+“directoryName” : string
+```
+Define target directory
+
+
+
+```json
+“type” : [suggest, suggestAll]
```

Comment:
I may be wrong but `type` property is already explained above, and `select` is a supported type for directory. @ataillefer ?
No space.

Reply:
Yes you're right.
So the `"type"` section should be removed here, and [there](https://github.com/nuxeo/doc.nuxeo.com-content/pull/837/files/db72e6d3c1acc70535d61c943a359d49ee7f5713#diff-2e4a1a7d60e55662ee0bf93298b9ac45R163) where it is defined it should be generic:

"Allows to search any of the available values (user, group, directory entry, etc.) without initially displaying the set of entries."